src/UrlGeneratorInterface.php

```diff
@@ -45,13 +45,11 @@ public function generateAbsolute(
         string $host = null
     ): string;
 
+    public function generateCurrent(array $replacedParams, string $fallbackRouteName = null): string;
```

Comment:
How about adjusting name a bit?

```suggestion
    public function generateFromCurrent(array $replacedParams, string $fallbackRouteName = null): string;
```

Reply:
Done